client/post-editor/editor-drawer/index.jsx

```diff
@@ -245,8 +246,11 @@ const EditorDrawer = React.createClass( {
 		}
 
 		const { plan } = this.props.site;
+		const { blog_public } = this.props.siteSettings;
+		const isSitePrivate = parseInt( blog_public, 10 ) !== 1;
```

Comment:
OK. To keep this PR moving, let's keep it, so at worst we're being overly careful, which is fine.